backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -35,7 +36,15 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
 
   def retryable: Boolean
 
-  def isFatal(throwable: Throwable): Boolean = throwable.isInstanceOf[CromwellFatalExceptionMarker]
+  // The scala package object (scala/package.scala) contains a neat list of runtime errors that are always going to be fatal.
```

Comment:
I'm confused as to exactly what file you mean (I'm assuming it's in the scala github repo?)